src/smartpeak/source/core/SequenceSegmentProcessors/StoreQuantitationMethods.cpp

```diff
@@ -60,9 +60,30 @@ namespace SmartPeak
 
   void StoreQuantitationMethods::getFilenames(Filenames& filenames) const
   {
-    filenames.addFileName("quantitationMethods", constructFilename("quantitationMethods.csv", static_filenames_), "Quantitation Methods", false, false, true);
+    filenames.addFileName(
+      "quantitationMethods", 
+      constructFilename("quantitationMethods.csv", static_filenames_),
+      "Quantitation Methods",
+      false,
+      false,
+      true);
   };
 
+  bool StoreQuantitationMethods::onFilePicked(const std::filesystem::path& filename, ApplicationHandler* application_handler)
+  {
+    if (application_handler->sequenceHandler_.getSequenceSegments().size() == 0)
+    {
+      LOGE << "File cannot be written without first loading the sequence.";
+      return false;
+    }
+    auto& sequence_segment = application_handler->sequenceHandler_.getSequenceSegments().at(0);
```

Comment:
How is one able to store quantitationMethods from different sequenceSegments?

Reply:
Yes, actually it cannot support multiple segments at the moment. indeed. I would need to think about it, how to make it possible to the user to select the sequence segment or maybe he will have to select a folder instead.

Reply:
Can you make a ticket for this?